lib/Extension/LanguageServerPhpstan/LanguageServerPhpstanExtension.php

```diff
@@ -119,8 +121,7 @@ public function configure(Resolver $schema): void
             self::PARAM_TMP_FILE_DISABLED => 'Disable the use of temporary files when.'
                 . ' This prevents as-you-type diagnostics, but ensures paths in phpstan config are respected.'
                     . ' See https://github.com/phpactor/phpactor/issues/2763',
-            self::PARAM_EDITOR_MODE => 'Use the editor mode of Phpstan https://phpstan.org/user-guide/editor-mode'
-                . ' (Requires phpstan 2.14 or higher)',
+            self::PARAM_EDITOR_MODE => 'DEPRECATED. Editor mode of Phpstan is used automatically when it\'s supported.'
```

Comment:
cc @mamazu removed this as not sure it makes sense now. we either detect the phpstan version or not and not sure if there's any benefit in not using editor  mode if it's available.